ipfs-pinning-service.yaml

```diff
@@ -449,6 +454,16 @@ components:
           summary: Example of multiple CIDs
           value: [QmbWqxBEKC3P8tqsKc98xmWNzrzDtRLMiMPL8wBuTGsMnR,bafkreigtdgsgv2f3bkhsmxvku3bpnnqzubcxeupf7fff5f7l7tlm2v237a]   # ?cid=Qm,bafy
 
+    name:
```

Comment:
This needs to be added to the `GET /pins` params list.

Reply:
Good catch, fixed in  49aab56